excludes.txt

```diff
@@ -1,17 +1,6 @@
 # TODO cryptic PCT-only errors: https://github.com/jenkinsci/bom/pull/251#issuecomment-645012427
 hudson.matrix.AxisTest
 
-# TODO Remove this exclusion when 2.387.x is no longer an active line
-# Refer to https://github.com/jenkinsci/blueocean-plugin/pull/2505
-# PR is included in blueocean-rest-impl versions for Jenkins 2.401.3 and later
-io.jenkins.blueocean.rest.impl.pipeline.MultiBranchTest
-
-# TODO https://github.com/jenkinsci/blueocean-plugin/pull/2471
-io.jenkins.blueocean.service.embedded.PipelineApiTest
```

Comment:
still seems to fail

Reply:
Thanks!  I've pushed f145660f756d525ad933e36f3abb630479bdeec2 that restores the exclusion.  Also pushed 6a982b9b72298b69c10debe9f5e11a5699235b83 to test the incremental build of the backport.